app/src/ui/delete-branch/delete-branch-dialog.tsx

```diff
@@ -63,15 +120,60 @@ export class DeleteBranch extends React.Component<
     )
   }
 
+  private renderLocalUnmergedWarning(unmergedCommits: number) {
+    if (this.state.existsOnRemote || this.state.localUnmergedCommits <= 0) {
+      return null
+    }
+
+    return (
+      <div>
+        <strong>
+          Warning: This branch has {unmergedCommits} unmerged{' '}
+          {unmergedCommits === 1 ? 'commit' : 'commits'}.
```

Comment:
```suggestion
          {unmergedCommits === 1 ? 'commit' : 'commits'} when compared to <Ref>{this.props.defaultBranch.name}</Ref>.
```

^ May cause linting error.. this is just a suggestion to communicate requested change.. needed tested. :)